airflow/www/views.py

```diff
@@ -3796,11 +3796,11 @@ def historical_metrics_data(self):
             data = {
                 "dag_run_types": {
                     **{dag_run_type.value: 0 for dag_run_type in DagRunType},
-                    **{run_type: sum_value for run_type, sum_value in dag_run_types},
+                    **dict(dag_run_types),
```

Comment:
```suggestion
                    **dag_run_types,
```

The additional dict is totally unnecessary. Same for the change below.